scalajvm/app/global/Options.scala

```diff
@@ -0,0 +1,10 @@
+package global
```

Comment:
Why is `Options` controller in `global` package? Does the `global` package have a special meaning in Play Framework like `Global` object? I think I'm missing something here...